cpp/ql/src/change-notes/2024-10-15-wrong-type-format-argument.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Fixed false positives in the `cpp/wrong-type-format-argument` ("Wrong type of arguments to formatting function") query if there are extraction errors in the function.
```

Comment:
This is technically not true, as there are valid cases where a function is implicitly defined and correctly being used as a argument in a printf call.

Reply:
Yes, it might also reject true-positives. Do you have a suggested text I could use?

Reply:
I think we mostly want to be explicit about what we ignore, i.e., calls to functions that are implicitly declared, as in the majority of cases we see that these are due to extraction errors, and not because the codebase is pre-C99 C-code, which allows implicit function declarations.